dist/render.js

```diff
@@ -0,0 +1,29 @@
+class Renderer {
+    constructor() {
+        this.resultsTemplate = Handlebars.compile($("#results-template").html())
+        this.statsTemplate = Handlebars.compile($("#stats-template").html())
+    }
+
+    renderResults(players) {
+        $("#results-container").empty()
+        const resultsHTML = this.resultsTemplate(players)
+        $("#results-container").append(resultsHTML)
+    }
+
+    doItNice(str) {
+        let words = str.split('_')
+        // words.forEach(w => { w = (w[0].toUpperCase() + w.slice(1))})
+        return words.join(' ');
+    }
```

Comment:
if you use this method, then this could be in a `utils.js` file (that way you can use it from anywhere, also from the Roster class, if they have the same functionalities).
if you don't use this method - then delete it...